app/survey_config/business_config.py

```diff
@@ -18,7 +21,10 @@ class BusinessSurveyConfig(
     footer_links: Iterable[MutableMapping] = field(
         default_factory=lambda: [
             Link(lazy_gettext("What we do"), "#").__dict__,
-            Link(lazy_gettext("Contact us"), "#").__dict__,
+            Link(
+                lazy_gettext("Contact us"),
+                f"{base_url}/contact-us/",
```

Comment:
We should reuse the link from the object, so `self.contact_us_url`.

Reply:
Reusing the survey config one now, thanks.